platform/src/main/java/org/stellar/anchor/platform/callback/RestCustomerIntegration.java

```diff
@@ -111,7 +111,7 @@ public void deleteCustomer(String id) {
     Response response = call(httpClient, callbackRequest);
     String responseContent = getContent(response);
 
-    if (response.code() != HttpStatus.OK.value()) {
+    if (response.code() != HttpStatus.NO_CONTENT.value()) {
```

Comment:
💡  Maybe support both, just in case? I mean, if we get a 2xx, that's usually interpreted as a successful response.